### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by bzy-debug**
Update react-tour content to be consistent with current API
see #1014

### CODE REVIEW

**Comment by brillout** [comment]
Thanks!

File: docs/pages/react-tour.page.server.mdx [hunk 1]
```diff
@@ -75,8 +75,6 @@ async function render(pageContext) {
       </body>
     </html>`;
 }
- 
-export const passToClient = ["pageProps"]
```
Comment (bzy-debug) [inline docs/pages/react-tour.page.server.mdx]: Is this line really unnecessary?

I get an error without this line:

截屏2023-07-12 15 50 47

And the button is not interactive.

Correct me if I miss anything :)

Comment (brillout) [reply docs/pages/react-tour.page.server.mdx]: You're right, it's required. But https://vite-plugin-ssr.com/react-tour#render-control is about showcasing render control and the code snippets don't aim to be complete. The `passToClient` part is actually explained down below at https://vite-plugin-ssr.com/react-tour#data-fetching.

Comment (bzy-debug) [reply docs/pages/react-tour.page.server.mdx]: Make sense, thanks for your explanation!

**Pull request merged**